File: API/Entity/Flight.cs
using System.ComponentModel.DataAnnotations;

namespace API.Entity
{
    public class Flight
    {
        [Required]
        public int FlightId { get; set; }
        [Required]
        public String FlightNo { get; set; }

        public DateTime Arrival { get; set; }   
        public DateTime Departure { get; set; }
        [Required]
        public String Destination { get; set; }
        
        //Create FK for AircaftTypes
        public int? AircraftTypeId { get; set; }
        
        public AircraftType AircraftType { get; set; }

        public virtual ICollection<Order>? Orders { get; set; }
    }
}

File: client/src/app/_models/flight.ts
import { DatePipe } from "@angular/common";
import { AircraftType } from "./aircraftType";

export class Flight {
    flightId: number;
    flightNumber: string;
    arrival: Date;
    departure: Date;
    destination: string;
    aircraftTypeId: number;
    aircraftType: AircraftType;
}